CHANGELOG.md

```diff
@@ -6,63 +6,104 @@ This changelog track changes to the qoqo aqt project starting at version 0.3.0
 
 ## 0.8.0
 
-* Updated pkg to account for new AQT API
+### Added in 0.8.0
+
```

Comment:
0.8.0 is the current version, this should be 0.9.0

Reply:
Either the new code goes into an `Unreleased` section, or in a new release. Either works for me